### PR TITLE
OCD-4247

### PR DESCRIPTION
**Pull request by mattystank**

### CODE REVIEW

File: src/app/pages/subscriptions/confirm-subscription-wrapper.jsx [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
+  Box,
+} from '@material-ui/core';
 import ChplConfirmSubscription from './confirm-subscription';
-
 import AppWrapper from 'app-wrapper';
 
 function ChplConfirmSubscriptionWrapper({ hash }) {
   return (
     <AppWrapper>
+      <Box sx={{ backgroundColor: '#f9f9f9'}}>
```
Comment (andlar) [inline src/app/pages/subscriptions/confirm-subscription-wrapper.jsx]: Nothing should go in these wrapper files that we want to keep. Eventually, all the wrapper files will be deleted, and if we just delete this one now this box and whatever it's doing will be lost. This should be in the "confirm-subscription" file somewhere

File: src/app/pages/subscriptions/confirm-subscription-wrapper.jsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { string } from 'prop-types';
-
+import {
+  Box,
```
Comment (andlar) [inline src/app/pages/subscriptions/confirm-subscription-wrapper.jsx]: This should have given you a bunch of warnings. I'm not sure of the specific language but:

- Box isn't used
- The imports from `prop-types` and `./confirm-subscription` shouldn't be in the same group
- Neither should the ChplConfirmSubscription and the AppWrapper

File: src/app/pages/subscriptions/manage-subscription.jsx [hunk 3]
```diff
@@ -130,113 +202,165 @@ function ChplManageSubscription(props) {
...
+        <Box className={classes.chplLeftHandFormat}>
+          <Card>
+            <Box className={classes.mySubsciptionImagery} />
+            <CardContent>
+              <Typography gutterBottom variant="h4" component="h2"><strong>Welcome to your subscription page!</strong></Typography>
+              <Typography>Here, you can easily view and manage your subscriptions.</Typography>
+            </CardContent>
+          </Card>
+          <Box>
+            <Box display="flex" flexDirection="row" justifyContent="space-between" pb={4} alignItems="center">
```
Comment (andlar) [inline src/app/pages/subscriptions/manage-subscription.jsx]: Any particular reason these attributes aren't in a style? They're all (almost) css attributes, right?

Comment (mattystank) [reply src/app/pages/subscriptions/manage-subscription.jsx]: Yes, let me fix this

**Pull request merged**